resources/views/settings/index.blade.php

```diff
@@ -284,12 +284,16 @@
                                         @elseif($setting->type == "file" && isset( $setting->value ))
                                             <div class="fileType">{{ $setting->value }}</div>
                                         @endif
+                                        @if($setting->type == "image")
+                                            <input type="file" name="{{ $setting->key }}" accept="image/*" />
```

Comment:
If you're going to do this, the if should only be around the difference (`accept` attribute), not duplicating the entire input tag.